src/PlayerManager.cpp

```diff
@@ -556,6 +558,26 @@ void PlayerManager::setSpecialPlayer(CNSocket* sock, CNPacketData* data) {
         std::cout << "\tSetValueType: " << setData->iSetValueType << std::endl;
         std::cout << "\tSetValue: " << setData->iSetValue << std::endl;
     )
+        //Handle serverside value-changes
+        switch (setData->iSetValueType) {
+        case(1):
+            plr.plr->HP = setData->iSetValue;
+            break;
+        case(2):
```

Comment:
```suggestion
        case 2:
```
Avoid parentheses in case statements.